dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/configMLDT/configForEutaxonomyFinancialsMLDT.ts

```diff
@@ -45,6 +47,25 @@ function getSpecifiedDataPointGetterFactory(
   return getDataPointGetterFactory(path, { ...sampleField, label: fieldLabel }, formatter);
 }
 
+/**
+ * The function defines the trigger for the shouldDisplay property
+ * @param displayValue value of the field which is checked
+ * @returns boolean to trigger the shouldDisplay condition
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function setShowIf(displayValue: any): boolean {
+  return !!(displayValue && displayValue != NO_DATA_PROVIDED);
```

Comment:
Checking for NO_DATA_PROVIDED makes no sense here. Since we plug in actual data from the backend.

Reply:
Weird. Hussain added that yesterday to fix some issues with the toggle button and it seemed to have done the trick.